scope.go

```diff
@@ -35,7 +35,7 @@ var nextScopeID = uint64(time.Now().UnixNano())
 
 type scope struct {
 	startTime   time.Time
-	id          scopeID
+	id          string
```

Comment:
if you change id type `scopeID` to `string` you need update all the references in tests.
`cannot use newScopeID() (value of type scopeID) as type string in struct literal`